methods.py

```diff
@@ -7,15 +7,6 @@
 from collections.abc import Mapping
 from typing import Iterator
 
-# We need to define our own `Action` method to control the verbosity of output
-# and whenever we need to run those commands in a subprocess on some platforms.
-from SCons import Node
-from SCons.Script import Action
-from SCons.Script import ARGUMENTS
-from SCons.Script import Glob
-from SCons.Variables.BoolVariable import _text2bool
-from platform_methods import run_in_subprocess
-
```

Comment:
For the record I moved those where they're used so that importing a pure Python function from `methods.py` doesn't require SCons.